CMakeLists.txt

```diff
@@ -38,6 +38,18 @@ set(KWIVER_BINARY_DIR           "${CMAKE_BINARY_DIR}")
 set(KWIVER_CONFIG_FILE          "${KWIVER_BINARY_DIR}/kwiver-config.cmake")
 # Create initial setup script
 set(KWIVER_SETUP_SCRIPT_FILE    "${KWIVER_BINARY_DIR}/setup_KWIVER.sh" )
+if(WIN32)
+  set(KWIVER_SETUP_BATCH_FILE    "${KWIVER_BINARY_DIR}/setup_KWIVER.bat" )
+  if(CMAKE_CXX_COMPILER_ID MATCHES MSVC)
+   if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 19.10)
+      set(_vcVersion vc15)
+    elseif(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 19)
+      set(_vcVersion vc14)
+    else()
```

Comment:
@dstoup Is there any plan for fletch to know that, if built with msvc, it can copy those pesky opencv dlls libs in to the install/bin?

If not, it seems that all users of fletch will have to write this special kind of code on windows to make sure all the dlls can be found